src/commands/backup.rs

```diff
@@ -221,9 +232,14 @@ impl BackupCmd {
                 }
             }
 
+            // use the correct source-specific run-before and run-after
+            let run_before = opts.run_before.clone();
+            let run_after = opts.run_after.clone();
+
             // merge "backup" section from config file, if given
             opts.merge(config.backup.clone());
 
+            run_before.run(&format!("run-before backup {source}"))?;
```

Comment:
I feel like `rustic` executing commands could almost always do an `info` log.
To me, because the commands executed depend on both the user and `rustic`, the user should be informed of what rustic does with the provided commands.
In the `CommandInput` type for instance, or before calling the command.

Reply:
Oh - this is always a discussion between people wanting more and people wanting less verbosity... 
I wouldn't log in `info` in the `CommandInput` as in `rustic_core` there might be situations where such a command is called quite often...
Currently this is a `trace` - we could discuss if `debug` is better suited..

Reply:
`debug` could be a good compromise indeed

Reply:
It is now `debug`